README.md

```diff
@@ -1,2 +1,166 @@
+
+> Your placement in the program will be based on reviews of your abstract.
+This should be a roughly 500 word outline of your presentation.
+This outline should concisely describe software of interest to the SciPy community, tools or techniques for more effective computing, or how scientific Python was applied to solve a research problem.
+A traditional background/motivation, methods, results, and conclusion structure is encouraged but not required.
+Links to project websites, source code repositories, figures, full papers, and evidence of public speaking ability are encouraged.
+
+#### Introduction and Motivation
+
+In experimental high energy physics (HEP), statistical fitting tools have been implemented almost entirely in C++.
+One common example used by experimentalists at CERN uses the "HistFactory" p.d.f. template.
+Prominently used in the 2012 discovery of the Higgs boson by the ATLAS and CMS collaborations, the p.d.f. template has only existed within the C++ based ROOT framework developed and maintained by CERN.
+pyhf is a pure-Python implementation of that statistical model for multi-bin histogram-based analysis with asymptotic interval estimation, and part of the Scikit-HEP project (https://scikit-hep.org/) ecosystem.
+pyhf supports modern computational graph libraries as computational backends in order to make use of features such as auto-differentiation and GPU acceleration.
+Additionally, the statistical models are defined in an expressive JSON schema, readily enabling preservation and distribution through services such as the Durham High-Energy Physics Database (HEPData) (https://www.hepdata.net/).
+
+#### Performance gain through tensorization
+
+Through adoption of open source "tensor" computational Python libraries, pyhf decreases the abstractions between a physicist analyst and the statistical modeling without sacrificing computational speed.
+In fact, through transforming computational graph complexity to tensor dimensionality, pyhf outperforms the traditional C++ implementations on data from real LHC analyses.
+pyhf's default computational backend is built from NumPy and SciPy, and supports TensorFlow, PyTorch, and JAX as alternative backend choices.
+These alternative backends support hardware acceleration on GPUs, and in the case of JAX JIT compilation, as well as auto-differentiation allowing for calculating the full gradient of the likelihood function --- all significantly speeding up fits.
+
+#### Model specification
+
+pyhf's model specification is in JSON as it offers long term support, is human and machine readable, and is highly compressible.
+All of these benefits make it ideal for analysis archival, and as almost every language has a JSON parser allows for the model specification to be independent of the HistFactory implementation language.
```

Comment:
```suggestion
All of these benefits make it ideal for analysis archival and, since almost every language has a JSON parser, allows for the model specification to be independent of the HistFactory implementation language.
```